web/src/test/java/com/replica/RouterServerTest.java

```diff
@@ -307,9 +340,9 @@ public void testWalkQuery() {
     public void testAutoQueryWithAlternatives() {
         final RouterOuterClass.StreetRouteReply response = routerStub.routeStreetMode(AUTO_REQUEST_WITH_ALTERNATIVES);
 
-        // we route with 2 auto profiles and combine results, and each profile should have produced at least one
+        // we route with 4 auto profiles and combine results, and each profile should have produced at least one
         // alternate
-        Predicate<Long> perProfilePathCountPredicate = pathCount -> pathCount > 1L;
+        Predicate<Long> perProfilePathCountPredicate = pathCount -> pathCount >= 1L;
```

Comment:
FYI @bryanwilliams1025 , I updated this predicate because with the latest graphhopper version bump `car_highway` is only returning 1 route here. Seems like >= 1 route per profile is still correct (even w/ alternatives turned on), so I think this should still be fine

Reply:
Hmm...agree >=1 is still correct, but are you surprised that GraphHopper no longer finds alternate routes for `car_highway`, even though the profile knobs, [very generous alternate route settings](https://github.com/replicahq/graphhopper/blob/998dd705b0e0bba4f5ad9b65eb7576b775892d9d/web/src/test/java/com/replica/RouterServerTest.java#L144-L146), and OSM are still the same? Especially since the origin and dest are nearly 20 miles apart. Realize this sort of thing is tricky to debug, but I'm wondering if it could relate to some of the quality differences in that [slack thread](https://replicahq.slack.com/archives/CK358RL0Z/p1673728078572269)

Reply:
Did some more testing and I'm kinda confused as to what might be going wrong 🤔 all of the custom profiles in the unit tests return >1 route, with only `car_freeway` returning 1 route. On the other hand, `car_freeway` is by far the most restrictive custom profile, really lowering the speed on certain kinda-big road types to force traffic onto highways.

Obviously, though, something has changed between the last two "updated" graphhopper commits we're pointing to, because with `b62dfbac89a4a79eb5bab305707c528e7bef35f5` we were getting >1 route for `car_freeway`

Reply:
Huh, ok...if I comment out the most restrictive clause in `freeway_car_custom_model.json` and re-run the alt route unit test, we suddenly get 5 routes returned instead of 1: 
```
    {
      "if": "road_class == PRIMARY || road_class == SECONDARY || road_class == TERTIARY || road_class == RESIDENTIAL || road_class == SERVICE",
      "multiply_by": "0.2"
    },
```

My best guess is that something internal changed that means we need to adjust our custom models (annoying but doable). I'll try to verify the sandbox is working now so we can do that, but also would like to confirm this hypothesis and perhaps still have the backup plan in place to fork off of `b62dfbac89a4a79eb5bab305707c528e7bef35f5` and apply the perf fixes directly there (seems annoying but probably possible (?))